src/glossary.rst

```diff
@@ -443,13 +443,21 @@ An :dt:`attribute macro` is a :t:`procedural macro` that consumes two streams
 of :t:`[token]s` to produce a stream of tokens, and defines a new :t:`outer
 attribute` that can be attached to :t:`[item]s`.
 
+.. _fls_24iVIlHhvnVO:
+
+auto trait
+^^^^^^^^^^
+
+:dp:`fls_d84nTOR4pZq5`
+An :dt:`auto trait` is ??? HELP LUKAS!
```

Comment:
Auto traits are traits that are automatically implemented for every type, unless the type in question contains at least one type that does not implement the trait or if the type opts-out of the implementation of the trait. (This opt-out is unstable syntax via `impl !Foo for Type {}`, which the std/core lib makes use of.

Basically this part here https://doc.rust-lang.org/beta/unstable-book/language-features/auto-traits.html#automatic-trait-implementations

Reply:
Now defined.